src/tests.rs

```diff
@@ -203,6 +203,144 @@ fn create_pool_test() -> Result<(), ContractError> {
     Ok(())
 }
 
+#[test]
+fn create_pool_test_1() -> Result<(), ContractError> {
+    let mut deps = mock_dependencies();
+    let info = mock_info("admin0", &[]);
+    let _ = instantiate(
+        deps.as_mut(),
+        mock_env(),
+        info.clone(),
+        InstantiateMsg {
+            event_tracker: Addr::unchecked("tracker0".to_string()),
+            deadline: 1000,
+        },
+    )?;
+    let _ = execute(
+        deps.as_mut(),
+        mock_env(),
+        info.clone(),
+        ExecuteMsg::RegisterJobId {
+            chain_id: Uint256::from(0u8),
+            job: "create_pool".to_string(),
+            job_id: "create_pool_eth".to_string(),
+        },
+    )?;
+    let _ = execute(
+        deps.as_mut(),
+        mock_env(),
+        info.clone(),
+        ExecuteMsg::RegisterJobId {
+            chain_id: Uint256::from(1u8),
+            job: "create_pool".to_string(),
+            job_id: "create_pool_bnb".to_string(),
+        },
+    )?;
+    let res = execute(
+        deps.as_mut(),
+        mock_env(),
+        info.clone(),
+        ExecuteMsg::CreatePool {
+            chain0_id: Uint256::from(0u8),
+            chain1_id: Uint256::from(0u8),
+            token0: "0x0000000000000000000000000000000000000001".to_string(),
+            token1: "0x0000000000000000000000000000000000000000".to_string(),
+            chain0_init_depositor: "0x0000000000000000000000000000000000000001".to_string(),
+            chain1_init_depositor: "0x0000000000000000000000000000000000000001".to_string(),
+            fee: 3000,
+        },
+    )?;
+    assert_eq!(res.messages.len(), 2);
+    let mut iter = res.messages.iter();
+    while let Some(msg) = iter.next() {
+        if let CosmosMsg::Custom(data) = msg.msg.clone() {
+            assert_eq!("create_pool_eth".to_string(), data.job_id);
+        }
+    }
+    let mut env = mock_env();
+    env.block.time = env.block.time.plus_seconds(2000u64);
+    let _ = execute(
```

Comment:
I think the `let _` is fine for this execute, but it seems like we should be asserting aspects of the created pool. Perhaps this is part of another test? If so we can merge the two.